### PR TITLE
Remove management permission

### PR DESCRIPTION
**Pull request by lesleyjanenorton**
This PR removes the `management` permission as requested in https://mozilla-hub.atlassian.net/browse/ADDONSOPS-274.

### CODE REVIEW

File: src/manifest.json [hunk 1]
```diff
@@ -35,7 +35,6 @@
       "captivePortal",
       "identity",
       "idle",
-      "management",
```
Comment (pdehaan) [inline src/manifest.json:38]: LOL, I was curious about this so I checked [the docs](https://www.devdoc.net/web/developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/management/uninstallSelf.html):
> This API does not require the "management" [API permission](https://www.devdoc.net/web/developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/permissions.html).

&hellip; but then realized there was a lengthy discussion in The Jira ticket about this.

**Review by pdehaan — APPROVED** [review]
Yeah... the code looks good to me (although admittedly not my area of expertise).

I'm _assuming_ we don't want any of the other optional parameters on the async `uninstall()`? (ie: gently requesting uninstall vs forced uninstall)

> - `showConfirmDialog` (Optional) &mdash; Boolean. If `showConfirmDialog` is `true`, the browser will show a dialog asking the user to confirm that the add-on should be uninstalled. Defaults to `false`.
> - `dialogMessage` (Optional) &mdash; String. An extra message that will be displayed in the confirmation dialog.

**Comment by pdehaan** [comment]
Also not 100% sure if the CircleCI and tests GitHub checks failing is a concern.

```
  0 passing (6s)
  9 failing
```

secure-proxy 77 `Failed to fetch config.yml file.`

**Comment by lesleyjanenorton** [comment]
Thanks @pdehaan . No additional params needed with `uninstallSelf()`.

**Pull request merged**